doc/nrf/security/crypto/crypto_supported_features.rst

```diff
@@ -1024,8 +1029,8 @@ The following table shows the Kconfig options for requesting Oberon PSA Crypto t
      - Not supported
      - Supported
      - Supported
-   * - SRP-6 password hashing
-     - :kconfig:option:`CONFIG_PSA_WANT_ALG_SRP_6`
+   * - SRP password hashing
+     - :kconfig:option:`CONFIG_PSA_WANT_ALG_SRP_PASSWORD_HASH`
```

Comment:
@frkv , CONFIG_PSA_WANT_ALG_SRP_6 was duplicated, so I edited it like this. Is this ok or we should keep a separate SRP-6 password hashing entry with a duplicated usage of this Kconfig?